index.html

```diff
@@ -251,11 +251,11 @@ <h3 class="skillsTitle">Languages</h3>
             <p class="skillsListTitle">JavaScript</p>
           </li>
           <li class="skillsDescList">
-            <img src="images/html5.svg" alt="javaScript" />
+            <img src="images/html5.svg" alt="html" />
             <p class="skillsListTitle">HTML</p>
           </li>
           <li class="skillsDescList">
-            <img src="images/css3.svg" alt="javaScript" />
+            <img src="images/css3.svg" alt="css" />
```

Comment:
Kindly change the contrast to be at least `4.5` of the `see project` buttons in the hover, because they failed in the contract check
![image](https://user-images.githubusercontent.com/24703171/155302019-d7f44445-32e1-41bd-9b6b-afbb51430efd.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)

Reply:
Thank you for the fast reply! The colors I've used are the colors from the Figma template and I am not allowed to change them because I need to follow them, so, because of this, I can't change the contrast, it has to be like this.

Reply:
![contrast](https://user-images.githubusercontent.com/90456900/155306213-3ede96d3-fe00-4d67-9678-c4e2db916d99.png)
 

As you can see, that's the color of the button.

![contrastText](https://user-images.githubusercontent.com/90456900/155306445-39fb6cb9-c826-4e1c-b4e6-35263f84a548.png)


And that's the color of the text.

Reply:
You can change the color on `hover` but try to choose a color that is similar as much as possible to Figma's design.

Reply:
I tried, but it still doesn't meet the requirements to pass. I need to change it to somewhere around blue, which will not be similar to the Figma design.

Reply:
Yes, blue is perfect.

Reply:
Done.

Reply:
Good job.